cdap-ui/app/features/tracker/controllers/results-ctrl.js

```diff
@@ -150,6 +150,9 @@ class TrackerResultsController {
         createDate: entity.metadata.SYSTEM.properties['creation-time'],
         datasetType: entity.metadata.SYSTEM.properties.type
       });
+      if(entity.metadata.SYSTEM.tags.indexOf('explore') !== -1) {
```

Comment:
@anwar6953 Right now the logic which we use to find if a dataset is explorable or not is wrong in CDAP. we should use this method in CDAP to find if its explorable. Its the otherway :)